test/helpers/expectThrow.js

```diff
@@ -1,7 +1,14 @@
-async function expectThrow (promise) {
+async function expectThrow (promise, message) {
   try {
     await promise;
   } catch (error) {
+    if (message) {
+      assert(
+        error.message.search(message) >= 0,
+        'Expected \'' + message + '\', got \'' + error + '\' instead',
+      );
+      return;
+    }
```

Comment:
Could you change this brace for an `else`, to emphasize the fact that the `message` parameter changes the usual behavior?